### PR TITLE
LPD-25772 Fix client-extension-web playwright tests

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

**Comment by izaera** [comment]
ci:test:playwright

**Review by markocikos — APPROVED** [review]
@izaera LGTM! :sparkles:

**Comment by izaera** [comment]
ci:test:sf

**Comment by izaera** [comment]
ci:test:playwright

File: modules/test/playwright/tests/client-extension-web/pages/EditClientExtensionsPage.ts [hunk 1]
```diff
@@ -33,6 +34,10 @@ export class EditClientExtensionsPage {
 			name: 'Publish',
 		});
 
+		this.descriptionCKEditor = page.locator(
+			'#cke__com_liferay_client_extension_web_internal_portlet_ClientExtensionAdminPortlet_description'
+		);
+
```
Comment (dsanz) [inline modules/test/playwright/tests/client-extension-web/pages/EditClientExtensionsPage.ts:40]: As spoken this morning, this is kind of fixing the symptom and not the root cause of elements not being available. We will tackle this in depth [here](https://liferay.atlassian.net/browse/LPD-25816)

**Comment by izaera** [comment]
Since there are no unique failures I'm forwarding this

**Comment by izaera** [comment]
ci:forward

**Pull request closed without merging**